CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ project(umock_c)
 
 set(UMOCK_C_VERSION 1.1.19)
 
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Do we really need this, as it might still be an issue for IoT folks.... We can also pull it out again when they come complaining though.